### PR TITLE
Code quality fixing

### PR DESCRIPTION
**Pull request by KhadarRashid**

### CODE REVIEW

**Comment by claraj** [comment]
@abdiawali please review

File: convert_state.py [hunk 1]
```diff
@@ -34,7 +34,8 @@ def main():
             print('try again')
 
 
-def convert_state_to_abbreviation(dictionary ):
+def convert_state_to_abbreviation(dictionary):
+    """Function to get state name and give abbreviation"""
     user_input = input('Enter state name: ').  capitalize()
```
Comment (abdiawali) [inline convert_state.py]: there is space between the . and the capitalize()

Comment (KhadarRashid) [reply convert_state.py]: fixed.

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,59 @@
...
 def main():
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+    """Reading in json data"""
 
-    print(data)
+    with open('data.json', 'r') as file:
+        state_json = file.read()
+        abbr_to_names = json.loads(state_json)
```
Comment (claraj) [inline convert_state.py:11]: Avoid abbreviations in variables names, use abbreviation_to_names. If I know the name is abbreviated, is it abbr_to_names or abrv_to_names or ? Autocomplete helps with longer names so it isn't more typing.

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,59 @@
...
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
-
-
-    #loop
+    # Create a dictionary of states (abbreviations, and a dictionary of abbreviations)
+    state_abbr = abbr_to_names
+    state_dict = {}
```
Comment (claraj) [inline convert_state.py:15]: Be more specific - names_to_abbreviations?

**Review by claraj — CHANGES_REQUESTED** [review]
Use .gitignore to ignore .idea files 

Code looks much better! Just a few small things to take a look at.

**Pull request closed without merging**